unbalanced_dataset/under_sampling/random_under_sampler.py

```diff
@@ -14,35 +14,35 @@
 class RandomUnderSampler(UnderSampler):
     """Class to perform random under-sampling.
 
-    Object to under sample the majority class(es) by randomly picking samples
+    Under-sample the majority class(es) by randomly picking samples
     with or without replacement.
 
     Parameters
     ----------
     ratio : str or float, optional (default='auto')
-        If 'auto', the ratio will be defined automatically to balanced
-        the dataset. Otherwise, the ratio will corresponds to the number
+        If 'auto', the ratio will be defined automatically to balance
+        the dataset. Otherwise, we use the ratio of the number
```

Comment:
I don't like this phrasing ("we use") since that the ratio is defined. If you can reformulate such that "the ratio is defined as" or something similar.